test/integration/022_bigquery_test/models/sql_header_model.sql

```diff
@@ -0,0 +1,12 @@
+{# This will fail if it is not extracted correctly #}
+{% call set_sql_header(config) %}
+  	CREATE TEMPORARY FUNCTION a_to_b(str STRING)
+	RETURNS STRING AS (
+	  CASE
+	  WHEN LOWER(str) = 'a' THEN 'b'
+	  ELSE str
+	  END
+	)
```

Comment:
I think you might just need a semicolon here - I see
```
Completed with 1 error and 0 warnings:

Database Error in model sql_header_model (models/sql_header_model.sql)
  Syntax error: Unexpected keyword CREATE at [12:3]
  compiled SQL at target/run/test/sql_header_model.sql
```

in the CI build

Reply:
Should be fixed!